new-dti-website/components/team/MemberGroup.tsx

```diff
@@ -33,7 +33,7 @@ const MemberSummary: React.FC<MemberSummaryProps> = ({
       <img
         src={image}
         alt={`${firstName}-${lastName}`}
-        className={`rounded-md h-auto ${enlarged ? 'w-[244px]' : 'w-[202px]'}`}
+        className={`rounded-md ${enlarged ? 'h-[244px] w-[244px]' : 'h-[202px] w-[202px]'} object-cover`}
```

Comment:
Crop the image at 244px by 244px in enlarged view, 202px by 202px in regular view.